WebRoot/js/zimbraMail/abook/controller/ZmHabTreeController.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Web Client
+ * Copyright (C) 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2016 Synacor, Inc.
+ *
+ * The contents of this file are subject to the Common Public Attribution License Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at: https://www.zimbra.com/license
+ * The License is based on the Mozilla Public License Version 1.1 but Sections 14 and 15
+ * have been added to cover use of software over a computer network and provide for limited attribution
+ * for the Original Developer. In addition, Exhibit A has been modified to be consistent with Exhibit B.
+ *
+ * Software distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTY OF ANY KIND, either express or implied.
+ * See the License for the specific language governing rights and limitations under the License.
+ * The Original Code is Zimbra Open Source Web Client.
+ * The Initial Developer of the Original Code is Zimbra, Inc.  All rights to the Original Code were
+ * transferred by Zimbra, Inc. to Synacor, Inc. on September 14, 2015.
+ *
+ * All portions of the code are Copyright (C) 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2016 Synacor, Inc. All Rights Reserved.
+ * ***** END LICENSE BLOCK *****
+ */
+
+/**
+ * @overview
+ * This file contains the address book tree controller class.
+ * 
+ */
+
+/**
+ * Creates an address book tree controller.
+ * @class
+ * This class is a controller for the tree view used by the address book 
+ * application. This class uses the support provided by {@link ZmOperation}. 
+ *
+ * @author Parag Shah
+ * 
+ * @extends		ZmFolderTreeController
+ */
+ZmHabTreeController = function() {
+	ZmFolderTreeController.call(this, ZmOrganizer.HAB);
+};
+
+ZmHabTreeController.prototype = new ZmFolderTreeController;
+ZmHabTreeController.prototype.constructor = ZmHabTreeController;
+
+ZmHabTreeController.prototype.isZmHabTreeController = true;
+ZmHabTreeController.prototype.toString = function() { return "ZmHabTreeController"; };
+
+// Public methods
+
+/**
+ * Shows the controller and returns the resulting tree view.
+ * 
+ * @param	{Hash}	params		 a hash of parameters
+ * @return	{ZmTreeView}	the tree view
+ */
+ZmHabTreeController.prototype.show =
```

Comment:
do we need this method, it just calls super class method

Reply:
Fixed.